### PR TITLE
Feat(Cortex XSOAR/content): Add SEKOIA.IO integration

### PR DESCRIPTION
**Pull request by PierrickV**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Relevant docker PR: https://github.com/demisto/dockerfiles/pull/8125

**Comment by MosheEichler** [comment]
@PierrickV It seems that unrelated files were accidentally entered. Please fix it

File: Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py [hunk 1]
```diff
@@ -0,0 +1,684 @@
...
+        "aliases",
+        "goals",
+        "revoked",
+        "created",
+        "modified",
+        "more_info",
+    ]
+    indicator_id = ""
+    markdown = ""
+    # Read every items of the reponse
```
Comment (MosheEichler) [inline Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py:371]:
```suggestion
    # Read every items of the response
```

**Review by MosheEichler — CHANGES_REQUESTED** [review]
Hi @PierrickV.
You did an Excellent Job!! :)

Please check the following notes.
And let me know when you are done.

Thank you.

File: Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/command_examples [hunk 2]
```diff
@@ -0,0 +1,3 @@
+!GetObservable value="eicar@sekoia.io" type="email-addr"
```
[the remark this answers is not in the document]
Comment (PierrickV) [reply Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/command_examples:1]: ok

File: Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py [hunk 3]
```diff
@@ -0,0 +1,684 @@
...
+    Extract hashes composing the STIX indicator pattern
+    """
+    hashes = {"sha256": "", "sha1": "", "md5": ""}
+
+    # Remove enclosing brackets
+    pattern_str = pattern_str.replace("[", "").replace("]", "")
+    stix_object_hashes = pattern_str.split("OR")
+    for object_hash in stix_object_hashes:
+        if "SHA-256" in object_hash:
+            hashes["sha256"] = object_hash.split("=")[1].strip(" ").strip("'")
```
[the remark this answers is not in the document]
Comment (PierrickV) [reply Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py:158]: Yes !

File: Packs/SEKOIAIntelligenceCenter/.secrets-ignore [hunk 4]
```diff
@@ -0,0 +1,26 @@
+eicar@sekoia.io
+eicar@sekoia.io
+eicar@sekoia.io
+eicar@sekoia.io
+eicar@sekoia.io
```
Comment (MosheEichler) [inline Packs/SEKOIAIntelligenceCenter/.secrets-ignore:5]:
```suggestion
```
Please remove.
And all duplicates.

File: Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/README.md [hunk 5]
```diff
@@ -0,0 +1,475 @@
+Fetch Indicator and Observables from SEKOIA.IO Intelligence Center.
+To use this integration, please create an API Key with the right permissions.
+
+This integration was integrated and tested with version 2.20220712 of SEKOIAIntelligenceCenter
```
Comment (MosheEichler) [inline Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/README.md:4]:
```suggestion
This integration was integrated and tested with version 2.20220712 of SEKOIA Intelligence Center
```

**Review by MosheEichler — COMMENTED** [review]
Perfect!

Please remove this file
Packs/SEKOIAIntelligenceCenter/pack_metadata.json.lock.
And go over the few final comments.

Thanks again you did an amazing job :)

File: Packs/SEKOIAIntelligenceCenter/.secrets-ignore (hunk 4 above)
Comment (PierrickV) [reply Packs/SEKOIAIntelligenceCenter/.secrets-ignore]: Fixed, sorry for this mistake :smile:

**Review by MosheEichler — APPROVED** [review]
Nice work

**Pull request merged**